### PR TITLE
Feature flags in crate's docs.rs documentation

### PR DESCRIPTION
**Pull request by rustworthy**
This is  to improve the docs as per the [comment](https://github.com/jonhoo/faktory-rs/pull/45#pullrequestreview-1807790758).

This change is [Reviewable](https://reviewable.io/reviews/jonhoo/faktory-rs/46)

### CODE REVIEW

File: src/proto/mod.rs [hunk 1]
```diff
@@ -15,9 +15,6 @@ pub use self::single::{
     Ack, Fail, Heartbeat, Info, Job, JobBuilder, Push, QueueAction, QueueControl,
 };
 
-// responses that users can see
-pub use self::single::Hi;
```
Comment (rustworthy) [inline src/proto/mod.rs:19]: not used outside of proto::single crate

File: tests/real/enterprise.rs [hunk 2]
```diff
@@ -360,4 +359,14 @@ fn ent_unique_job_bypass_unique_lock() {
...
     }
+
+    // let's consume three times from the queue to verify that the first two jobs
+    // have been enqueued for real, while the last one has not.
+    let mut c = ConsumerBuilder::default();
+    c.register("order", |j| -> io::Result<_> { Ok(eprintln!("{:?}", j)) });
+    let mut c = c.connect(Some(&url)).unwrap();
+
+    assert!(c.run_one(0, &[queue_name]).unwrap());
+    assert!(c.run_one(0, &[queue_name]).unwrap());
```
Comment (rustworthy) [inline tests/real/enterprise.rs:370]: to address [comment](https://github.com/jonhoo/faktory-rs/pull/45#discussion_r1443988164)

**Review by jonhoo — APPROVED** [review]
Excellent, thank you!

**Comment by jonhoo** [comment]
Released in 0.12.4 :tada:

**Pull request merged**